src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -42,7 +42,7 @@ public FunctionsController(WebScriptHostManager scriptHostManager, WebHookReceiv
 
             // Authorize the request
             SecretManager secretManager = (SecretManager)controllerContext.Configuration.DependencyResolver.GetService(typeof(SecretManager));
-            HttpBindingMetadata httpFunctionMetadata = (HttpBindingMetadata)function.Metadata.InputBindings.FirstOrDefault(p => p.Type == BindingType.HttpTrigger);
+            HttpTriggerBindingMetadata httpFunctionMetadata = (HttpTriggerBindingMetadata)function.Metadata.InputBindings.FirstOrDefault(p => p.Type == BindingType.HttpTrigger);
```

Comment:
Isn't http always intrinsically a trigger? What's the different between the two?